indexer/packages/postgres/src/db/migrations/migration_files/20240131142122_create_compliance_status_table.ts

```diff
@@ -0,0 +1,36 @@
+import * as Knex from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema
+    .createTable('compliance_status', (table) => {
+      table.string('address').primary(); // dYdX V4 chain address
+      table.enum(
+        'status', // compliance of the address
+        [
+          'COMPLIANT', // the address is compliant
+          'FIRST_STRIKE', // the address has a single-strike
+          'CLOSE_ONLY', // the address is in close-only mode
+          'BLOCKED', // the address is blocked
+        ],
+      ).notNullable();
+      table.enum(
+        'reason',
+        [
+          'MANUAL', // the address was manually set to the status
+          'US_GEO', // the address was set to the status due to connecting from US (restricted geography)
+          'CA_GEO', // the address was set to the status due to connecting from CA (restricted geography)
+          'SANCTIONED_GEO', // the address was set to the status due to connection from sanctioned geography
+          'COMPLIANCE_PROVIDER', // the address was set to the status due to being flagged by a compliance provider
+        ],
+      ).nullable().defaultTo(null); // null for COMPLIANT addresses
+      table.timestamp('createdAt').notNullable();
```

Comment:
nit: Default to current timestamp like [here](https://github.com/dydxprotocol/v4-chain/blob/main/indexer/packages/postgres/src/db/migrations/migration_files/20230915173306_update_compliance_data_updated_at_to_default.ts) and make `createdAt` nullable in the create object?

Reply:
done